readme.md

```diff
@@ -1,17 +1,18 @@
 # sbt-scalafix
 
 This is the sbt plugin for [Scalafix](https://scalacenter.github.io/scalafix/),
-a refactoring and linting tool for Scala.
-For instructions on how to install the plugin, refer to the [website](https://scalacenter.github.io/scalafix/docs/users/installation).
+a refactoring and linting tool for Scala. For instructions on how to install the
```

Comment:
Sorry for the noisy diff, I have "format on save" enabled and just realized after creating the PR 😅